packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -367,14 +410,177 @@ export class AllocationReceiptCollector implements ReceiptCollector {
     return this.models.vouchers.findAll({
       where: { protocolNetwork: this.protocolNetwork },
       order: [['amount', 'DESC']], // sorted by highest value to maximise the value of the batch
       limit: this.voucherRedemptionMaxBatchSize, // limit the number of vouchers to the max batch size
     })
   }
 
+  //TODO: Consider adding new configurations for RAV similar to vouchers
+  // (voucherRedemptionThreshold, voucherRedemptionMaxBatchSize, voucherRedemptionBatchThreshold)
+  startRAVProcessing() {
+    const notifyAndMapEligible = (signedRavs: ValidRavs) => {
+      if (signedRavs.belowThreshold.length > 0) {
+        const logger = this.logger.child({})
+        const totalValueGRT = formatGRT(
+          signedRavs.belowThreshold.reduce(
+            (total, signedRav) =>
+              total.add(BigNumber.from(signedRav.rav.rav.valueAggregate)),
+            BigNumber.from(0),
+          ),
+        )
+        logger.info(`Query RAVs below the redemption threshold`, {
+          hint: 'If you would like to redeem vouchers like this, reduce the voucher redemption threshold',
+          voucherRedemptionThreshold: formatGRT(this.voucherRedemptionThreshold),
+          belowThresholdCount: signedRavs.belowThreshold.length,
+          totalValueGRT,
+          allocations: signedRavs.belowThreshold.map(
+            (signedRav) => signedRav.rav.rav.allocationId,
+          ),
+        })
+      }
+      return signedRavs.eligible
+    }
+
+    const pendingRAVs = this.getPendingRAVsEventual()
+    const signedRAVs = this.getSignedRAVsEventual(pendingRAVs)
+    const eligibleRAVs = signedRAVs
+      .map(notifyAndMapEligible)
+      .filter((signedRavs) => signedRavs.length > 0)
+    eligibleRAVs.pipe(async (ravs) => await this.submitRAVs(ravs))
+  }
+
+  protected getPendingRAVsEventual(): Eventual<RavWithAllocation[]> {
+    return joinEventual({
+      timer: timer(30_000),
+      allocations: this.allocations,
+    }).tryMap(
+      async ({ allocations }) => {
+        const ravs = await this.pendingRAVs()
+        return ravs
+          .map((rav) => {
+            const domain = {
+              chainId: this.protocolNetwork.split(':')[1],
+              name: 'TAP',
+              version: '1',
+              verifyingContract: this.escrowContracts!.tapVerifier.address,
+            }
+            const signedRav = rav.getSignedRAV()
+            console.log({ domain, rav })
+            const signerAddress = utils.verifyTypedData(
+              domain,
+              {
+                ReceiptAggregateVoucher: [
+                  { name: 'allocationId', type: 'address' },
+                  { name: 'timestampNs', type: 'uint64' },
+                  { name: 'valueAggregate', type: 'uint128' },
+                ],
+              },
+              signedRav.rav,
+              signedRav.signature,
+            )
+            console.log({ signerAddress, ravSigner: rav.senderAddress })
```

Comment:
Remove this console.log and maybe convert this to an assert to check if the senderAddress is correct for that allocationId. You may need to find some way to get this information from the subgraph.